pyfarm/agent/service.py

```diff
@@ -774,4 +823,5 @@ def callback_agent_id_set(
                 "`%s` was %s, adding system event trigger for shutdown",
                 key, change_type)
 
-            self.scheduled_tasks.start()
+            self.schedule_call(
+                config["agent_master_reannounce"], self.reannounce)
```

Comment:
Yep, you're right it's never reached in the tests.  It's probably the same issue I encountered in the tests for #215, which I discovered after updating the tests here, so I'll see if I can fix the tests so we do reach this line.